addons/account/views/account_move_views.xml

```diff
@@ -629,26 +635,27 @@
 
                                 <!-- Invoice payment terms (only invoices) + due date (only invoices / receipts) -->
                                 <label for="invoice_payment_term_id"
-                                       attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt'))]}"/>
-                                <div class="d-flex" attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt'))]}">
+                                    attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'in_refund', 'out_receipt', 'in_receipt'))]}"/>
```

Comment:
remove 'in_refund' & 'in_receipt' as well in all your `attrs` diff

Reply:
done